src/constants/categories.js

```diff
@@ -131,7 +131,8 @@ const categories:Array<Category> = [
         byline: "Police, law & fines",
         icon: icons.Legal,
         search: {
-            q: "law -(coordinating bodies)",
+            q: "legal -(coordinating bodies) -permits -ceremonies" +
```

Comment:
Need a space after `ceremonies` or before `-making` otherwise this will result in `-ceremonies-making`